src/components/Figure.astro

```diff
@@ -0,0 +1,9 @@
+---
+// https://docs.astro.build/en/guides/integrations-guide/markdoc/#custom-image-components
+// https://discord.com/channels/830184174198718474/1182681722215743578/1204466454922002452
+// Open Github issue: https://github.com/withastro/astro/pull/11761
```

Comment:
This Github issue will be fixed in the next Github release so we won't need to hack around this